### ISSUE TITLE
Computation of best threshold

### ISSUE THREAD

**Issue by AntiLibrary5** [ZiyunLiang/IterMask2#2]
Hi,
Reading the paper, it is not clear how the `best_threshold` is computed. Then looking at `test.py`, we see there is a loop over the validation set which calls upon `validation_thres()` and saves certain files (`_mask_save.pt` and `_loss_save.pt`) which are then used for `iter_mask_refinement()`. My queries are:
1. How many val images are ideal. And should they be images from normal distribution or images with anomalies?
2. What computation are you exactly doing to compute the threshold?
3. If the arg `best_threshold` is True where we call `iter_mask_refinement_bestthres()`, then whats the difference with the previous approach?

Thank you.

**Comment by ZiyunLiang** [comment]
Thanks for your interest in our work.

1. The size of the validation set typically depends on the overall scale of the dataset, but using approximately 10% is generally sufficient. The validation samples come from the normal distribution, as they are intended to reflect the model’s reconstruction error on unseen normal tissue.
2. Please refer to the sensitivity analysis section of the paper, as written in the paper: 'We choose τ from the healthy validation set’s error map obtained by applying random masks in the same way as training.' 'τ is set at 80 percentile of the error map in BraTS and ISLES, and 70 in ATLAS'
3. please refer to the 'Human-AI collaboration' section of the paper in the exp section. The best threshold is used to simulate user selection. 'we tested our model by incrementally raising τ value during the shrinking process and selected the iteration with the optimal performance per image for evaluation, obtaining improved performance'

Thank you
